src/sidebar/util/time.js

```diff
@@ -63,18 +19,32 @@ let formatters = {};
 function clearFormatters() {
   formatters = {};
 }
+
+/**
+ * Calculate time delta in milliseconds between two `Date` objects
+ *
+ * @param {Date} date
+ * @param {Date} now
+ */
+function delta(date, now) {
+  return now - date;
+}
+
 /**
- * Efficiently return `date` formatted with `options`.
+ * Efficiently return date string formatted with `options`.
  *
- * This is a wrapper for Intl.DateTimeFormat.format() that caches
- * DateTimeFormat instances because they're expensive to create.
- * Calling Date.toLocaleDateString() lots of times is also expensive in some
+ * This is a wrapper for `Intl.DateTimeFormat.format()` that caches
+ * `DateTimeFormat` instances because they're expensive to create.
+ * Calling `Date.toLocaleDateString()` lots of times is also expensive in some
  * browsers as it appears to create a new formatter for each call.
  *
+ * @param {Date} date
+ * @param {Object} options - Options for `Intl.DateTimeFormat.format()`
+ * @param {Object} Intl - JS internationalization API implementation; this
+ *                      param is present for dependency injection during test.
  * @returns {string}
- *
  */
-function format(date, options, Intl) {
+function formatIntl(date, options, Intl) {
```

Comment:
I renamed this function because so many things in this module were named `format` before. I'm not wedded to the specific name as long as it isn't `format`.